apps/graphql/src/definitions/ScreenshotDiff.js

```diff
@@ -4,6 +4,12 @@ import { s3 as getS3, getSignedGetObjectUrl } from "@argos-ci/storage";
 import { ScreenshotLoader } from "../loaders";
 
 export const typeDefs = gql`
+  enum ScreenshotDiffType {
+    new
+    passing
+    update
+  }
```

Comment:
```suggestion
  enum ScreenshotDiffStatus {
    added
    stable
    updated
  }
```

For me it is more a status than a type. Also `added`, `stable`, `updated` looks better to me.